triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -490,7 +490,7 @@ object Trigger {
 }
 
 private[lf] class Runner private (
-    compiledPackages: CompiledPackages,
+    private[trigger] val compiledPackages: CompiledPackages,
```

Comment:
Rule simulator needs access to `compiledPackages` in order to define a Speedy machine instance